### ISSUE TITLE
Do you have an update plan available on .net core?

### ISSUE THREAD

**Issue by Jeongwonkyu** [hikalkan/scs#38]

**Comment by Vinz3056** [comment]
+1.
Very useful idea !

**Comment by gfazzola** [comment]
I think there is no simple and really cool option like this in .net core. 
It would really be an excellent alternative !!

**Comment by hikalkan** [comment]
@NeedJustWord has sent PR. Anyone wants to test it? I will merge if it works properly.

**Comment by gfazzola** [comment]
I'm going to take some time to try.

**Comment by SenerSoft** [comment]
Hi Halil,

I have been using your SCS as TCP Server for a long while within standard .Net framework.
I am really appreciated for SCS, it is exceptionally a well written TCP client/server.

And now, I have to move on .Net Core. I still want to use SCS. At a glance, I noticed that ASPNet Core doesn't have "**System.Runtime.Remoting.Proxies** and **System.Runtime.Remoting.Messaging**" anymore. It complains that accordingly.
Apparently "Real Proxy" has to be replaced by "Disptach Proxy" as Microsoft recommended.

Long story short question, by chance do you have .NET Core version of SCS? Or a suggestion for a solution or an alternative one it does as good as you developed SCS? 

Hoscakal,
Sener

**Comment by hikalkan** [comment]
Hi,

I definitely want to do this, but I don't have time currently. I would probably use the Castle DynamicProxy instead of Remoting.Proxies.

I don't know a good alternative. You may want to try signalr. But I love SCS since it is very simple to use, fast & stable.

**Comment by SenerSoft** [comment]
Thanks a lot for your reply, it is clear, understood. I would also love to contribute that but my main focus on the IoT devices recently. Therefore, until you have .Net Core version, I am going to try out "[NetCoreServer](https://github.com/chronoxor/NetCoreServer)" and see how they cope with the features you have. 

Thanks a lot again for the SCS anyway. It still works and it will work on my applications in some industries all over the world without a single hiccup.
